style.css

```diff
@@ -0,0 +1,342 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+.wrapper {
+	max-width: 930px;
+	min-width: 320px;
+	font-family: 'GothamPro', serif;
+	margin: 0 auto;
+	box-sizing: border-box;
+}
+
+/* header */
+
+.header {
+	width: 100%;
+	height: 63px;
+	display: grid;
+	flex-direction: row;
+	grid-template-columns: 0.7fr 4fr 1.5fr 0.6fr 0.6fr 0.3fr;
+	margin-bottom: 33px;
+	padding-top: 24px;
+	padding-bottom: 8px;
+}
+
+.header__logo {
+	display: flex;
+	align-items: center;
+}
+
+.header__logo-image {
+	width: 55px;
+	height: 55px;
+}
+
+.header__logo-image_adaptive {
+	display: none;
+}
+
+.header__menu {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	margin-right: 140px;	
+}
+
+.header__menu-bar {
+	display: none;
+}
+
+.menu-bar {
+	display: none;
+}
+
+.link {
+	text-decoration: none;
+	color: #000;
+	text-transform: uppercase;
+	font-size: 12px;
+	font-weight: bold;
+	line-height: 11px;
+}
+
+.header__search {
+	display: flex;
+	position: relative;
+	justify-content: center;
+	align-items: center;
+	width: 160px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)